### PR TITLE
AED Unit and Defib Belt update!

### PR DESCRIPTION
**Pull request by ShadzFen**
## About The Pull Request
It's finally here...

- Added AED Unit.
- Added AED Unit cabinet.
- Added Defib belt.
- Added new combat mechanics additionally to the already existing combat mechanics, that allow overcharging the defibrillator unit and shocking a target for life-threatening damage (don't worry, I tried to balance it as best as possible. It's not BF4 style).
- Fixed the medical belt and champion belt not displaying on mob when worn.

Details. **I'm gonna try to keep it spoiler free.**

- Overcharging requires the unit to be hacked. This can be done in two ways. One of which requires a certain level of COG stat.
- Overcharging has a cooldown to avoid insta-killing. 
- Overcharging requires much more cell charge than standard defibbing.
- Removing the existing cell requires a screwdriver, instead of dragging and dropping like with flashlights.
- When not overcharged, but with the safety disabled, the unit can be used for a softer combat mechanic. Faster than overcharging, but not even close to the amount of damage.
- The defib can revive people who are dead for less than 15 minutes, have all body parts and have been medically treated to remain stable once alive. The defib will say what state the patient is in, and why a defib attempt was failed or successful.
- The AED Unit specifically, has a built in audio tutorial! This allows non-medical crew to be able to use the unit without having medical training.

**ErisDefib.dmi**
image

**Changes to mob.dmi**
image

SleNBNm

## Why It's Good For The Game
Adds new gameplay features and allows revival of players who have been medically treated and prepped for revival.

## Changelog
```changelog
add: AED Unit
add: Defib belt unit.
add Overcharge mechanic for defib units.
add: A lot of sprites related to the defibs. (Courtesy of Michiyamenotehifunana)
fix: Medical and champion belt now have on-mob sprites.
```

### CODE REVIEW

File: code/__DEFINES/items_clothing.dm [hunk 1]
```diff
@@ -112,6 +112,7 @@
 // Inventory slot strings.
 // since numbers cannot be used as associative list keys.
 //icon_back, icon_l_hand, etc would be much better names for these...
+#define slot_belt_str		"slot_belt"
```
Comment (Michiyamenotehifunana) [inline code/__DEFINES/items_clothing.dm:115]: Is this bit actually still used, considering we figured out the icon_override thing?

File: code/controllers/configuration.dm [hunk 2]
```diff
@@ -446,7 +446,7 @@ GLOBAL_LIST_EMPTY(storyteller_cache)
 
 				if("ipr_minimum_age")
 					config.ipr_minimum_age = text2num(value)
-			
+
```
Comment (Michiyamenotehifunana) [inline code/controllers/configuration.dm:449]: It seems that this file wasn't actually changed, but the whitespace is being screwy. Harmless, but it'd probably be better if the changes to this were reverted somehow.

**Review by Michiyamenotehifunana — COMMENTED** [review]
Looks great, there's just two little bits that seem a extraneous tho.

File: code/__DEFINES/items_clothing.dm (hunk 1 above)
Comment (ShadzFen) [reply code/__DEFINES/items_clothing.dm:115]: Yeah I saw that, but I couldn't remember if we used it or not. If we didn't I'll remove it.

Comment (Michiyamenotehifunana) [reply code/__DEFINES/items_clothing.dm]: A quick look through shows that nothing uses it, so yeah.

File: zzz_modular_syzygy/defiblocker.dm [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+	var/open = FALSE
+	var/obj/item/device/defib_kit/defib
+
+/obj/structure/defibcabinet/update_icon()
+	icon_state = initial(icon_state)
+	if(!open)
+		icon_state = "[icon_state]"
+		return
+	if(defib)
```
Comment (martinlyra) [inline zzz_modular_syzygy/defiblocker.dm:17]:
```suggestion
	else if (defib)
```

**Review by martinlyra — CHANGES_REQUESTED** [review]
Just one nitpick. And there are merge conflicts in the `cev-eris.dme` due to me merging other files.

**Review by martinlyra — APPROVED** [review]

**Pull request merged**